snapcraft/tests/test_commands.py

```diff
@@ -14,25 +14,15 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-import fixtures
 
-from unittest import mock
-
-from snapcraft import help
+# Snapcraft modules
 from snapcraft import tests
+from snapcraft import commands
```

Comment:
The common way to do multiple imports from the same module is:

from snapcraft import (
    commands,
    tests
)